src/server/static/openapi.yaml

```
@@ -147,17 +147,44 @@ paths:
     get:
       tags:
         - stock
-      summary: Chart [Beta]
-      description: Get historical stock prices [Beta]
+      summary: Chart
+      description: Get historical stock prices
+      parameters:
+        - in: query
+          name: from
+          schema:
+            type: string
+          required: true
+          description: Start date of returned range in YYYY-MM-DD format.
+        - in: query
+          name: to
+          schema:
+            type: string
+          required: true
+          description: End date of returned range in YYYY-MM-DD format.
+        - in: query
+          name: aggregate
+          schema:
+            type: integer
+          required: false
+          description: Number of days to aggregate data. Default is 1.
```

We don't use periods in the other descriptions, so let's keep it that way:
```suggestion
      parameters:
        - in: query
          name: from
          schema:
            type: string
          required: true
          description: Start date of returned range in YYYY-MM-DD format
        - in: query
          name: to
          schema:
            type: string
          required: true
          description: End date of returned range in YYYY-MM-DD format
        - in: query
          name: aggregate
          schema:
            type: integer
          required: false
          description: Number of days to aggregate data (default is 1)
```